src/FSharp.Configuration/ProvidedTypes.fs

```diff
@@ -941,7 +942,9 @@ type ProvidedSymbolType(kind: SymbolKind, args: Type list) =
         | SymbolKind.Array _ -> typeof<System.Array>
         | SymbolKind.Pointer -> typeof<System.ValueType>
         | SymbolKind.ByRef -> typeof<System.ValueType>
-        | SymbolKind.Generic gty  -> ProvidedSymbolType.convType args gty.BaseType
+        | SymbolKind.Generic gty  -> 
+            if gty.BaseType = null then null else
+            ProvidedSymbolType.convType args gty.BaseType
```

Comment:
Sorry that was not obvious for me. Is there a reason to not reference the corresponding nuget package in this case? Can you also briefly explain (or send me a link) why we even need to copy them?